students/merrix/session03/mymailroom.py

```diff
@@ -0,0 +1,101 @@
+import sys  # imports go at the top of the file
+from collections import defaultdict
+from operator import itemgetter
+
+donor_db = defaultdict(list)
+
+
+donor_db = {"William Gates":[653772, 12.17],
+            "Jeff Bezos": [877.33],
+            "Paul Allen" : [663.23, 43.87, 1.32],
+            "Mark Zuckerberg": [1663.23, 4300.87, 10432.0]
+           }
+
+prompt = "\n".join(("Welcome to James Dot Donations!",
+          "Please choose from below options:",
+          "1 - Send a Thank you",
+          "2 - Create a report",
+          "3 - Send Letter",
+          "4 - To Exit",
+          ">>> "))
+
+
+def view_donor_db():
+    print (donor_db.keys())
+    choose = input("enter a name from the list or a new name to donate: ").title()
+    if choose not in donor_db.keys():
+        add_donor_db(choose)
+    elif choose in donor_db.keys():
```

Comment:
Minor: No need to have `elif` with the negation of the `if` condition, because `else` alone implies the negation of the `if` condition anyway.